beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -631,3 +637,120 @@ func (bs *Server) validateEquivocation(blk interfaces.ReadOnlyBeaconBlock) error
 	}
 	return nil
 }
+
+// GetBlockRoot retrieves hashTreeRoot of ReadOnlyBeaconBlock/BeaconBlockHeader.
+func (bs *Server) GetBlockRoot(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "beacon.GetBlockRoot")
+	defer span.End()
+
+	afterPath, ok := strings.CutPrefix(r.URL.Path, "/eth/v1/beacon/blocks/")
```

Comment:
hmm is this how we've been doing it in other spots? i feel like there's an ability to find this specific position between / and /

Reply:
`mux.Vars(r)["block_id"]`